src/go/docker-credential-none/cmd/root.go

```diff
@@ -51,13 +52,22 @@ using the same format that docker uses when no credsStore field is specified in
 This helper is intended for testing purposes, but will be used on Linux systems
 unless 'pass' and/or 'secretservice' is available.`,
 		configFileName),
+	DisableSuggestions: true,
+	SilenceUsage:       true,
 }
 
 // Execute adds all child commands to the root command and sets flags appropriately.
 // This is called by main.main(). It only needs to happen once to the rootCmd.
 func Execute() {
 	err := rootCmd.Execute()
 	if err != nil {
+		ptn := regexp.MustCompile(`unknown command "(.*?)" for "(?i:docker-credential-none(?:\.exe)?)"`)
```

Comment:
This seems like it doesn't belong in this PR.

Reply:
Actually, the PR is incomplete. It needs a test against an unknown endpoint, and that the error message is "standard". Which is why I made the above change.

Reply:
So hold off on reviewing for now....